src/redux/books/books.js

```diff
@@ -0,0 +1,23 @@
+const ADD_BOOK = 'book-store/books/ADD_BOOK';
+const REMOVE_BOOK = 'book-store/books/REMOVE_BOOK';
+
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return state.filter((i) => i.id !== action.payload.id);
```

Comment:
Since you set the `id` to the `REMOVE_BOOK` payload you can access it like as `action.payload` no need to add `.id` as the end. Kindly, remove that.